app/src/main/java/com/dessalines/thumbkey/keyboards/CommonKeys.kt

```diff
@@ -33,6 +38,14 @@ val SETTINGS_KEY_ITEM =
             size = FontSizeVariant.LARGE,
             color = ColorVariant.SECONDARY,
         ),
+        swipeType = SwipeNWay.TWO_WAY_HORIZONTAL,
```

Comment:
Why not just get rid of the SETTINGS_KEY_ITEM entirely?

Reply:
I was going to delete it, but I thought I'd keep it as there's some PRs with layout updates, but you're right, it's never used and the others will be changed to the emoji key anyway :).